auth/auth/templates/account-creating.html

```diff
@@ -5,11 +5,9 @@
   <div class="spinner"></div>
   <b>Creating account for {{ username }}</b>
 </div>
-<script>
-  let protocol = location.protocol.replace("http", "ws")
-  let sock = new WebSocket(protocol + location.host + "{{ base_path }}/creating/wait");
-  sock.onmessage = function (event) {
-      window.location.reload()
-  }
+
+<input type="hidden" id="_base_path" value="{{ base_path }}" />
```

Comment:
We could probably move this and csrf into header `meta` elements. I think that would - 

1. Be more canonical than hidden input elements
2. Be available to all scripts and subpages equally
3. Not risk id-collision problems if we load more than one account-creating.html subpage at a time (I don't think that's likely here, but elsewhere we might do that)

In our top level page, something like - 
```html
<head>
  <meta name="base-path" content="{{ base_path }}">
</head>
```

And then in the javascript that needs it - 
```html
const basePathMeta = document.querySelector('meta[name="base-path"]');
const basePath = basePathMeta ? basePathMeta.content : '';
```

and the same for csrf.